alacritty_terminal/src/window.rs

```diff
@@ -167,6 +169,14 @@ impl Window {
         // Set OpenGL symbol loader. This call MUST be after window.make_current on windows.
         gl::load_with(|symbol| windowed_context.get_proc_address(symbol) as *const _);
 
+        // On X11, embed the window inside another if the parent ID has been set
+        #[cfg(not(any(target_os = "macos", windows)))]
+        {
+            if let Some(parent_window_id) = config.window.embed {
```

Comment:
It's not exposed, that it's on X11. You should add  `if event_loop.is_x11()` or something, since there's Wayland.

Reply:
K.  Added.